### DISCUSSION TITLE
Serving a Website or Shiny App at 0.0.0.0 | /en/2018/09/localhost-0-0-0-0/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1207] [Comments]
https://yihui.org/en/2018/09/localhost-0-0-0-0/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
xaringan::inf_mr() doesn't allow you to specify the host. I just pushed a new version (0.13.2) of the servr package to Github: <https://github.com/yihui/servr> With this version, you can set options([servr.host](http://servr.host) = '0.0.0.0') before calling xaringan::inf_mr().

> Originally posted on 2019-05-29 16:24:30

[a comment by an automated account is not shown]

**Comment by hlblade** [comment]
您好，在windows环境下，如何把shiny服务后台运行，写个bat脚本shiny::runApp(host = '0.0.0.0')，这样是可以运行的，但是这个命令窗口无法关闭。windows下也没有nohup，请教该如何处理？谢谢！

**Reply by yihui** [reply]
命令行通常可以在末尾添加 `&` 将其发送到后台运行，这对 Windows 和 Unix 应该都通用。不过为什么需要它在后台运行呢？发送到后台的弊端正是不方便关闭，你需要找到进程 ID 才能杀死它。
